src/gui/properties/trackerlistwidget.cpp

```diff
@@ -356,6 +356,37 @@ void TrackerListWidget::loadTrackers()
 
     loadStickyItems(torrent);
 
+    const auto toString = [](const BitTorrent::TrackerEntry::Status status) -> QString
```

Comment:
I think this can be moved to trackerentry.cpp file but I don't insist it being done in this PR.